.github/workflows/publish.yml

```diff
@@ -30,9 +30,13 @@ jobs:
         run: ./gradlew clean build
 
       - name: Publish artifacts
-        run: ./gradlew publish
+        run: ./gradlew jreleaserConfig publish jreleaserDeploy
```

Comment:
https://jreleaser.org/guide/latest/concepts/workflow.html

You can verify your configuration by running `./gradlew jreleaserConfig`

<img width="650" alt="스크린샷 2025-07-02 13 18 55" src="https://github.com/user-attachments/assets/bcc68639-2c4c-479d-8e45-3a0d1b5dae90" />